### DISCUSSION TITLE
Action/Cross filters for bar charts in superset

### DISCUSSION THREAD

**Discussion by Narendra678** [apache/superset#18367] [Q&A / Help]
Hi Team,

I am able to add cross/action filters for Treemap v2 and pie charts, but not for bar chart. 
Can't we enable cross filters to bar charts? please suggest.

Best regards,
Naren

**Comment by mauriziogucciardo1** [comment]
Hi Narendra678,
Unfortunately I don't have an answer to your question. But, I would be grateful if you help me about how insert/manage cross / action filters for Treemap v2 and pie charts?
Best regards,
Maurizio
